.circleci/config.yml

```diff
@@ -10,23 +10,12 @@ variables:
   install_sys_deps: &install_sys_deps
     run:
       name: install build-essential
-      command: apt-get update && apt-get install -y build-essential libz-dev libcurl3-dev
+      command: apt-get update && apt-get install -y build-essential libz-dev libcurl3-dev curl
   install_gitlfs: &install_gitlfs
     run:
       name: Install git-lfs
       command: conda install --yes -c conda-forge git-lfs && git lfs install
-  restore_cache: &restore_cache
```

Comment:
I'd keep the caching functionality when testing for all the models.

Reply:
Yes done.